shipstation_integration/orders.py

```diff
@@ -182,15 +183,15 @@ def create_erpnext_order(
 			continue
 
 		settings = frappe.get_doc("Shipstation Settings", store.parent)
-		item_code = create_item(item, settings=settings, store=store)
+		stock_item = create_item(item, settings=settings, store=store)
 		item_notes = get_item_notes(item)
 		so.append(
 			"items",
 			{
-				"item_code": item_code,
+				"item_code": stock_item.item_code,
 				"qty": item.quantity,
-				"uom": frappe.db.get_single_value("Stock Settings", "stock_uom"),
-				"conversion_factor": 1,
+				"uom": stock_item.sales_uom,
```

Comment:
@batonac We can't assume that the Sales UOM will be set. We can see if it exists, then fall back to the Item's Stock UOM and then the global default UOM (current code, not great, unlikely to fail, better than an error).

Reply:
Good point. I wrongly assumed that `sales_uom` was required ... should have checked that.

Reply:
`stock_uom` is required on an item, so I'm not sure why we'd ever need the global default UOM